### PR TITLE
get values from solar system

### PR DESCRIPTION
**Pull request by Romaaaaaaan**

### CODE REVIEW

File: lab5.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        '''
+        self.planets.append(planet)
+
+    def sort_by_length_of_day(self):
+        '''
+        Функція яка сортує планети за довжиною дня
+        '''
+        self.planets.sort(key=lambda x: x.length_of_day)
+        for i in self.planets:
+            print(f"Назва планети: {i.name}, Довжина світлового дня: {i.length_of_day}")
```
Comment (Kapitoshka-a) [inline lab5.py]: It would be better if you replace the print with a return and implement print in main file

File: lab5.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+    def find_distance_between(self, planet_a, planet_b):
+        '''
+        Фунція яка шукаю відстань між планетами
+        '''
+        print(abs(planet_a.distance_from_sun - planet_b.distance_from_sun))
```
Comment (Kapitoshka-a) [inline lab5.py]: This method should also make return

**Review by Kapitoshka-a — COMMENTED** [review]
Everything is fine exept that you print relust instead of returning it. Also you should add a main file with implementing all mogic there

File: lab5.py (hunk 2 above)
Comment (Romaaaaaaan) [reply lab5.py]: it has already done

File: lab5.py (hunk 1 above)
Comment (Romaaaaaaan) [reply lab5.py]: i can't do this

Comment (Romaaaaaaan) [reply lab5.py]: done

File: main2.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+'''
+імпортує класи з файлу lab5
+'''
+from lab5 import Planet, PlanetType, Planetary
+
+def println():
```
Comment (Kapitoshka-a) [inline main2.py]: unclear name for function.

**Review by Kapitoshka-a — APPROVED** [review]

File: main2.py (hunk 3 above)
Comment (Romaaaaaaan) [reply main2.py]: done

**Pull request merged**